ecosystem/typescript/sdk/src/token_client.ts

```diff
@@ -47,18 +43,31 @@ export class TokenClient {
     description: string,
     uri: string,
```

Comment:
Should it be possible to pass in a chain ID and not fetch it? Perhaps not relevant for this PR though.

Reply:
Yeah. This PR just tries to do a drop-in replacement for the JSON txn submissions. Tried to keep the interface the same till we do next major version bump.